src/backend/joanie/core/models/courses.py

```diff
@@ -323,7 +323,7 @@ def signature_backend_references_to_sign(self, **kwargs):
                 submitted_for_signature_on__isnull=False,
                 student_signed_on__isnull=False,
                 order__organization=self,
-                order__state=enums.ORDER_STATE_VALIDATED,
+                order__state=enums.ORDER_STATE_COMPLETED,
```

Comment:
I think once the contract has been signed by the student, it should be available to the organization signature. That means, instead of retrieving validated order, I'll only exclude cancelled ones.


```suggestion
                order__state=~Q(enums.ORDER_STATE_CANCELED),
```